pkg/repository/clone_mirror.go

```diff
@@ -15,6 +15,7 @@ package repository
 
 import (
 	"fmt"
+	"github.com/pingcap/tiup/pkg/colorutil"
```

Comment:
Please run `make check` before commit so that the `go fmt` is invoked automatically to format the code.